repair/row_level.cc

```diff
@@ -845,6 +845,17 @@ class repair_meta {
     }
 
 public:
+    future<> clear_gently() noexcept {
+        return do_for_each(_peer_row_hash_sets, [] (repair_hash_set& x) {
+            x.clear();
```

Comment:
I think this is quite large and needs to be cleared gently too.

Reply:
Done in v2 and v3.